pkg/core/system.go

```diff
@@ -411,7 +413,7 @@ func deleteAllKnative(kc *kubectlClient, resourceType string) error {
 func deleteCrds(kc *kubectlClient, suffix string) error {
 	fmt.Printf("Deleting CRDs for %s\n", suffix)
 	crdList, err := kc.kubeCtl.Exec([]string{"get", "customresourcedefinitions","-ocustom-columns=name:metadata.name", "--no-headers=true"})
-	if err != nil {
+	if err == nil {
```

Comment:
Are you sure about this!?

Reply:
yes, if we deleted one of the CRDs we can list the resources for it, so we get an error, should be able to just move on to the next resource type

Reply:
I'm not sure I follow. I assume if `err != nil` (the now only case where this code path will stay in this function), then crdList is undefined